src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar calendar__month-length--29 calendar__start-day--tue">
```

Comment:
`month-length` and `start-day` are not elements of `calendar`, it's modifiers, please change to  `calendar--month-length--29` / `calendar--start-day--tue`